src/libraries/System.Runtime.Extensions/tests/System/StringComparer.cs

```diff
@@ -86,8 +86,12 @@ public static IEnumerable<object[]> UpperLowerCasing_TestData()
             //                          lower                upper          Culture
             yield return new object[] { "abcd",             "ABCD",         "en-US" };
             yield return new object[] { "latin i",          "LATIN I",      "en-US" };
-            yield return new object[] { "turky \u0131",     "TURKY I",      "tr-TR" };
-            yield return new object[] { "turky i",          "TURKY \u0130", "tr-TR" };
+
+            if (PlatformDetection.IsIcuGlobalization)
```

Comment:
Actually can we change this to `PlatformDetection.IsNotInvariantGlobalization`? You will need to merge with master to get that. 

The reason for that is because this is also supported on Windows systems that don't have `ICU` and will end up running on NLS.

Reply:
Done.